tests/integration/clock.php

```diff
@@ -13,4 +13,18 @@ function testClock() {
     $this->assertRegex('/^c:\d+:\d+:\d+:\d+$/', $clock['clock'],
                        "looks clocky");
   }
+
+  function testClockSync() {
```

Comment:
would you mind adding this as a new test in the python test suite instead?  We want to move away from the php based tests.

You could add `test/integration/test_clock.py` with just the python equivalent of this part in it.  Take a look at `tests/integration/test_since.py` for an example.